src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -186,18 +190,13 @@ public void visitToken(DetailAST ast) {
                     brace = objBlock.getFirstChild();
                 }
                 break;
-            case TokenTypes.LITERAL_WHILE:
-            case TokenTypes.LITERAL_CATCH:
-            case TokenTypes.LITERAL_SYNCHRONIZED:
-            case TokenTypes.LITERAL_FOR:
-            case TokenTypes.LITERAL_TRY:
-            case TokenTypes.LITERAL_FINALLY:
-            case TokenTypes.LITERAL_DO:
-            case TokenTypes.LITERAL_IF:
-            case TokenTypes.STATIC_INIT:
-            case TokenTypes.LAMBDA:
```

Comment:
I do not expect removal of supported tokens, please explain this update.
they are still present in getAcceptableTokens so this is really weird. 
Regression testing will be required

Reply:
returned cases back